src/crate/client/cursor.py

```diff
@@ -210,3 +221,31 @@ def duration(self):
                 "duration" not in self._result:
             return -1
         return self._result.get("duration", 0)
+
+    def _convert_rows(self):
+        """
+        Iterate rows, apply type converters, and generate converted rows.
+        """
+        assert "col_types" in self._result and self._result["col_types"], \
+               "Unable to apply type conversion without `col_types` information"
+
+        # Resolve `col_types` definition to converter functions. Running the lookup
+        # redundantly on each row loop iteration would be a huge performance hog.
+        types = self._result["col_types"]
+        converters = [
+            self._converter.get(type) for type in types
+        ]
+
+        # Process result rows with conversion.
+        for row in self._result["rows"]:
+            yield [
+                convert(value)
+                for convert, value in zip(converters, row)
+            ]
+
+    @staticmethod
+    def get_default_converter(mappings: Optional[ConverterMapping] = None) -> Converter:
+        """
+        Return the standard converter instance.
+        """
+        return DefaultTypeConverter(more_mappings=mappings)
```

Comment:
Is this needed? It's just wrapping a constructor which could be called directly?

Reply:
It looks like `get_default_converter()` is currently never called with any parameters, so I guess it can also be removed. Thanks!